### PR TITLE
Create binder-dev.

### PR DESCRIPTION
**Pull request by hogabrie**
@samuelba My first try to create a -dev package. Can you take a look?

### CODE REVIEW

**Comment by hogabrie** [comment]
These are the generated debians. I checked locally, looks good to me :) 

[binder.zip](https://github.com/ANYbotics/binder/files/7944271/binder.zip)

File: cmake/binderConfig.cmake.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(pybind11)
+
+include("${CMAKE_CURRENT_LIST_DIR}/binderTargets.cmake")
+
+set_and_check(binder_INCLUDE_DIRS "@PACKAGE_INCLUDE_INSTALL_DIR@")
+
+check_required_components(binder)
```
Comment (samuelba) [inline cmake/binderConfig.cmake.in:10]:
```suggestion
check_required_components(binder)

set(binder_FOUND true)
```

for find_package()

**Comment by samuelba** [comment]
Otherwise looks good to me.

File: cmake/binderConfig.cmake.in (hunk 1 above)
Comment (hogabrie) [reply cmake/binderConfig.cmake.in:10]: I actually had to check, but seems  that I got lucky and this is done by the macro: https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html

Comment (samuelba) [reply cmake/binderConfig.cmake.in:10]: ah ok :+1:

**Pull request merged**